cmd/translate/translate.go

```diff
@@ -0,0 +1,17 @@
+package translate
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var translateCmd = &cobra.Command{
+	Use:   "translate",
+	Short: "This command sets configuration",
+	Long: `translate command sets configuration from:
+			environment variables
+			yaml file
+			CLI arguments`,
+	Run: func(cmd *cobra.Command, args []string) {},
+}
+
+func init() {}
```

Comment:
empty init() func?

Reply:
init( ) func was generated by cobra and for now it's empty because we don't have any configurations or flags that we have to add.